driver-core/pom.xml

```diff
@@ -16,55 +16,54 @@
 
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>com.datastax.cassandra</groupId>
         <artifactId>cassandra-driver-parent</artifactId>
-        <version>3.2.1-SNAPSHOT</version>
+        <version>3.3.0-SNAPSHOT</version>
     </parent>
+
     <artifactId>cassandra-driver-core</artifactId>
-    <packaging>bundle</packaging>
     <name>DataStax Java Driver for Apache Cassandra - Core</name>
-    <description>A driver for Apache Cassandra 1.2+ that works exclusively with the Cassandra Query Language version 3
+    <description>
+        A driver for Apache Cassandra 1.2+ that works exclusively with the Cassandra Query Language version 3
         (CQL3) and Cassandra's binary protocol.
     </description>
-    <url>https://github.com/datastax/java-driver</url>
+
+    <url>https://github.com/datastax/java-driver/driver-core</url>
```

Comment:
👍 on removing the URL tag, for individual modules the parent link is probably appropriate anyways.